packages/fx-core/resource/package.nls.json

```diff
@@ -522,6 +522,18 @@
   "core.common.ErrorFetchApiSpec": "Your OpenAPI description document should be accessible without authentication, otherwise download and start from a local copy.",
   "core.common.SendingApiRequest": "Sending API request: %s. Request body: %s",
   "core.common.ReceiveApiResponse": "Received API response: %s.",
+  "core.envFunc.unsupportedFile.errorLog": "\"%s\" is not a valid file. Supported format: %s.",
+  "core.envFunc.unsupportedFile.errorMessage": "Invalid file. Check output for details.",
```

Comment:
Do we want to say 'output panel' here?

Reply:
I have removed "Check output for details." as TTK will always add such sentence at the end of the error if running in VS.